WordPress/UITestsFoundation/Screens/CommentsScreen.swift

```diff
@@ -0,0 +1,63 @@
+import ScreenObject
+import XCTest
+
+public class CommentsScreen: ScreenObject {
+
+    private let replyTextViewGetter: (XCUIApplication) -> XCUIElement = {
+        $0.textViews["ReplyText"]
+    }
+
+    private let backButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.navigationBars.buttons["Reader"]
+    }
+
+    private let replyButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.buttons["Reply"]
+    }
+
+    var replyTextView: XCUIElement { replyTextViewGetter(app) }
+    var backButton: XCUIElement { backButtonGetter(app) }
+    var replyButton: XCUIElement { replyButtonGetter(app) }
+
+    init(app: XCUIApplication = XCUIApplication()) throws {
+        try super.init(
+            expectedElementGetters: [
+                // swiftlint:skip:next opening_brace
+                { $0.navigationBars["Comments"] },
+                replyTextViewGetter
+            ],
+            app: app,
+            waitTimeout: 7
+        )
+    }
+
+    public static func isLoaded() -> Bool {
+        (try? ReaderScreen().isLoaded) ?? false
+    }
+
+    public func navigateBack() throws -> ReaderScreen {
+        backButton.tap()
+        return try ReaderScreen()
+    }
+
+    @discardableResult
+    public func replyPost(_ comment: String) -> CommentsScreen {
+        app.otherElements.containing(.staticText, identifier: "Reply to post").lastMatch?.tap()
```

Comment:
if we add an accessibilityIdentifier to the element, we could simplify the call to something like this:
```suggestion
        app.otherElements["reply-to-post"].tap()
```

accessibility identifier line `self.replyTextView.accessibilityIdentifier = @"reply-to-post";` can be added after this line: 
https://github.com/wordpress-mobile/WordPress-iOS/blob/e540592299399b3ffb31c455196ba661c257ece4/WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsViewController.m#L747

this way i think if we wanted to test in a language other than English, we would be able to without depending on the text on the UI.

Reply:
Yeah, that would be much better! I'll have a separate PR for this. 🙇‍♂️